resotocore/resotocore/model/db_updater.py

```diff
@@ -140,7 +150,11 @@ async def merge_graph(self, db: DbAccess) -> GraphUpdate:  # type: ignore
             log.debug("Graph read into memory")
             builder.check_complete()
             graphdb = db.get_graph_db(nxt.graph)
+            outer_edge_db = db.get_pending_outer_edge_db()
             _, result = await graphdb.merge_graph(builder.graph, model, nxt.change_id, nxt.is_batch)
+            if nxt.task_id: # and len(builder.outer_edges) :
```

Comment:
only store if there are outer edges
```suggestion
            if nxt.task_id and builder.outer_edges:
```